tests/test_classifier/test_learning_curve.py

```diff
@@ -1,133 +0,0 @@
-# tests.test_classifier.test_learning_curve
```

Comment:
I was going to say that we need to put a deprecated warning on the learning curve visualizer, but since it wasn't in the documentation. We are probably okay with moving it and then documenting it.

Reply:
Yeh, pretty much all my previous PRs and upcoming PRs are going to be about documenting the undocumented classifiers and ensuring they have image comparison tests. I thought this would be simple, but as you can see it does require a fair amount of refactoring.

Reply:
flag anything that needs to go into `contrib` and I can migrate it over

Reply:
Will do